### PR TITLE
Fix up docker compatibility messages

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
LGTM

I assume the libpod.conf manpage is removed intentionally? A general question: as the files are renamed, wouldn't it makes to also `s/podman/docker/` in the manpages to have valid references across the manpages?

**Comment by rhatdan** [comment]
@vrothberg No the idea is to make sure people know they are using podman,  We don't want them to think they are using docker.  That would be considered dishonest.   We want them to know they are using podman, but if they cut and paste a command from google, or just do it by accident, or have scripts hard coded to run the Docker CLI, we want them to work without having to sed 's/docker/podman in their scripts.

**Comment by rhatdan** [comment]
their is no equivalent for libpod.conf in docker.

File: Makefile [hunk 1]
```diff
@@ -232,9 +232,7 @@ install.cni:
 install.docker: docker-docs
 	install ${SELINUXOPT} -D -m 755 docker $(BINDIR)/docker
 	install ${SELINUXOPT} -d -m 755 $(MANDIR)/man1
-	install ${SELINUXOPT} -d -m 755 $(MANDIR)/man5
```
Comment (TomSweeneyRedHat) [inline Makefile]: Do you really want to remove this?

Comment (rhatdan) [reply Makefile]: Yes, their are no man5 man pages.

Comment (TomSweeneyRedHat) [reply Makefile]: As @vrothberg pointed out, we're currently shipping libpod.conf.5.  Do we want to stop shipping that?

**Comment by mheon** [comment]
Code LGTM

File: Makefile (hunk 1 above)
Comment (rhatdan) [reply Makefile]: No.  That is not installed here, it is installed in the standard install.  This is just about installing links for Docker alias, so people accidently typing docker commands or man docker build, will get information back.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=mheon

**Comment by TomSweeneyRedHat** [comment]
LGTM, thx for the links 411

**Pull request closed without merging**